objects-api/src/main/kotlin/com/ritense/objectsapi/opennotificaties/OpenNotificatieService.kt

```diff
@@ -96,7 +96,9 @@ class OpenNotificatieService(
         val documentResult = documentService.createDocument(newDocumentRequest);
 
         if (documentResult.resultingDocument().isEmpty) {
-            logger.error { "Errors occurred during creation of dossier for productaanvraag: ${documentResult.errors()}" }
+            var logMessage = "Errors occurred during creation of dossier for productaanvraag:"
```

Comment:
Might have been nicer to use:
documentResult.errors().joinToString(separator="\n")